samples/unit-tests/no-data/members/demo.js

```diff
@@ -0,0 +1,15 @@
+QUnit.test("defaultOptions", assert => {
+    const {
+        noData: {
+            attr: {
+                zIndex
+            }
+        }
+    } = Highcharts.getOptions();
+
+    assert.equal(
+        zIndex,
+        1,
+        "should have defaultOptions.noData.attr.zIndex to equal 1 (#12343)"
```

Comment:
Better write as a natural sentence with upper case first: "Default z index should be 1 (#12343)"